notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb

```diff
@@ -78,14 +83,14 @@
         "\n",
         "In this tutorial, you create an AutoML image object detection model from a Python script, and then do a batch prediction using the Vertex AI SDK. You can alternatively create and deploy models using the `gcloud` command-line tool or online using the Cloud Console.\n",
```

Comment:
- change: "Vertex AI SDK"
- to: "Vertex AI SDK for Python"

- change: "You can alternatively create and deploy models..."
- to: "Alternatively, you can create and deploy models...

Reply:
addressed comments. Please verify